spring-cloud-azure-context/src/main/java/com/microsoft/azure/spring/cloud/context/core/impl/StorageConnectionStringBuilder.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.azure.spring.cloud.context.core.impl;
+
+import com.microsoft.azure.spring.cloud.context.core.api.Region;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class StorageConnectionStringBuilder {
+    private static final String DEFAULT_PROTOCOL = "DefaultEndpointsProtocol";
+
+    private static final String ACCOUNT_NAME = "AccountName";
+
+    private static final String ACCOUNT_KEY = "AccountKey";
+
+    private static final String ENDPOINT_SUFFIX = "EndpointSuffix";
+
+    private static final String HTTP_PROTOCOL = "http";
```

Comment:
Is default protocol `HTTPS`?

Reply:
I followed https://docs.microsoft.com/en-us/azure/storage/queues/storage-java-how-to-use-queue-storage#setup-an-azure-storage-connection-string. This link used `http`. I'll handle this issue in another PR.

Reply:
Sure. You can take a look at [here](https://docs.microsoft.com/en-us/azure/storage/common/storage-security-guide#encryption-in-transit).